content/_data/changelogs/weekly.yml

```diff
@@ -4809,6 +4809,52 @@
     # pull: 4128 too minor (Update last links from /people to /asynchPeople)
     # pull: 4129 (Run the Linux JDK 8 & 11 builds using ACI agents rather than Ubuntu VMs)
     # pull: 4142 (Test shouldn't be interactively breaking)
+- version: '2.190'
+  date: 2019-08-18
+  changes:
+    - type: bug
+      pull: 4152
+      issue: 23349
+      message: |-
+        Add support of emojis and other non-UTF-8 characters in job names. 🎉
+    - type: bug
+      pull: 4148
+      issue: 58595
+      message: |-
+        RSS and Atom feeds did not contain all necessary metadata. (regression in 2.186)
+    - type: rfe
+      pull: 3770
+      issue: 54772
+      message: |-
+        Expose real environment variables from an agent on the UI.
+    - type: rfe
+      pull: 4134
+      issue: 58734
+      message: |-
+        Use SHA-256 instead of MD5 for generating crumbs/CSRF tokens.
+    - type: rfe
+      pull: 4121
+      issue: 58157
+      message: |-
+        Truncate long build names on the UI to prevent alignment issues.
+    - type: rfe
+      pull: 4122
+      issue: 58571
+      message: |-
+        Developer: <code>AbstractItem#renameTo</code> now checks <code>#isNameEditable</code> before renaming.
+    # pull: 4116 too minor (Correct package name)
+    # pull: 4132 too minor (Replace Charset.forName with StandardCharsets)
+    # pull: 4136 too minor ([JENKINS-58736] Correction of /assets throwing 500)
+    # pull: 4140 too minor (Add Polish translations)
+    # pull: 4141 not in product (Updating LICENSE section of readme.md)
+    # pull: 4143 not in product (Clean up formatting)
+    # pull: 4144 too minor ([JENKINS-58836] Wrong spanish argentinian translation for 'logout')
+    # pull: 4147 too minor ([JENKINS-58892] - Remove build-pipeline-plugin from setup wizard)
```

Comment:
LTS candidate, so I would rather put it to the list

Reply:
2.176.3 is done, and the next baseline will be chosen next week, it's unlikely we'll choose a release that doesn't have this.